Sources/SwiftProtobuf/Google_Protobuf_Any+Extensions.swift

```diff
@@ -12,6 +12,8 @@
 ///
 // -----------------------------------------------------------------------------
 
+import Foundation
```

Comment:
Can you add a comment here that we need to keep this for Linux so we don't break things again by accident.